forum/forum/settings.py

```diff
@@ -158,3 +158,60 @@
 SECURE_CONTENT_TYPE_NOSNIFF = True
 SECURE_HSTS_SECONDS = 3600  
 SECURE_SSL_REDIRECT = os.getenv('DJANGO_SECURE_SSL_REDIRECT', 'True') == 'True'
+
+LOG_DIR = 'logs'
+os.makedirs(LOG_DIR, exist_ok=True)
+
```

Comment:
If access to the file system is restricted, os.makedirs() may throw an error. Add exception handling:
try:
    os.makedirs(LOG_DIR, exist_ok=True)
except OSError as e:
    raise RuntimeError(f"Failed to create log directory {LOG_DIR}: {e}")